components/Pet/Pet.js

```diff
@@ -106,16 +125,39 @@ export default function Pet({ petData }) {
         <p>
           <span>Type:</span> {petData.type}
         </p>
-        <StyledList>
-          <span>Characteristics:</span>
-          {petData.characteristics.map((characteristic, index) =>
-            index < petData.characteristics.length - 1 ? (
-              <li key={index}>{characteristic + ", "}</li>
-            ) : (
-              <li key={index}>{characteristic}</li>
-            )
-          )}
-        </StyledList>
+
+        {!onGraveyard && (
+          <StyledList>
+            <span>Characteristics:</span>
+            {petData.characteristics.map((characteristic, index) =>
+              index < petData.characteristics.length - 1 ? (
+                <li key={index}>{characteristic + ", "}</li>
+              ) : (
+                <li key={index}>{characteristic}</li>
+              )
+            )}
+          </StyledList>
+        )}
+
+        {!petData.alive && onGraveyard && (
+          <>
+            <p>
+              <span>Time of Death: </span>
+              {new Date(petData.timeOfDeath).toLocaleString()}
+            </p>
+            <ReviveButton onClick={() => onGraveyard(petData.id)}>
+              <Image src={reviveIcon} alt="Confirm Icon" width={40} />
+            </ReviveButton>
+          </>
+        )}
+
+        {petData.revived && (
+          <>
+            <p>
+              <span>Status:</span> Revived
+            </p>
+          </>
```

Comment:
There is no need for fragments here:

```suggestion
            <p>
              <span>Status:</span> Revived
            </p>
```